src/models/currency.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+
+const currencySchema = mongoose.Schema({
+
+    currencyName: {
+        type: String,
+        require: true,
+        unique: true
+    },
+
+    ratio: {
+        type: Number,
+        require: true,
+    },
+
+    availableAmount: {
+        type: Number,
+        require: true
+    },
+
+});
+
+//currencySchema.index({ userId: 1, email: 1 });
+
+const Currency = mongoose.model("Currency", currencySchema);
+
+module.exports = Currency;
```

Comment:
The file should always end with a new line
```suggestion
module.exports = Currency;

```